includes/definitions/discovery/bintec-beip-plus.yaml

```diff
@@ -0,0 +1,21 @@
+# Some values are hardcoded because snmpbulkwalk is not supported
+modules:
+    os:
+        version: BIANCA-BRICK-MIB::biboAdmSWVersion.0
+        serial: BIANCA-BRICK-MIB::biboABrdSerialNo.0.0
+        hardware: BIANCA-BRICK-MIBRES-MIB::cpuDescr.1.0
```

Comment:
not sure if there is a better one, but this doesn't really seems to be a hardware model

Reply:
Unfortunately I was no able to find a better string representing the hardware version. I also cannot see any hardware version information being logged into the device or even using its integrated SNMP browser. With the above config, it reads "Onboard MIPS 24Kec CPU". Would you rather remove this information than having this string displayed?

Reply:
Up to you if it makes sense or not!